python/gtsam/tests/test_Cal3Unified.py

```diff
@@ -139,6 +139,17 @@ def test_jacobian(self):
         self.gtsamAssertEquals(z, np.zeros(2))
         self.gtsamAssertEquals(H @ H.T, 4*np.eye(2))
 
+        Dcal = np.zeros((2, 10), order='F')
+        Dp = np.zeros((2, 2), order='F')
+        camera.calibrate(img_point, Dcal, Dp)
+        
+        self.gtsamAssertEquals(Dcal, np.array(
```

Comment:
numpy arrays should be tested with `np.testing.assert...`

Reply:
internally gtsamAssertEquals also deal with NP arrays